index.js

```diff
@@ -173,7 +173,8 @@ const onChange = (object, onChange, options = {}) => {
 				isPrimitive(value) ||
 				isBuiltinWithoutMutableMethods(value) ||
 				property === 'constructor' ||
-				options.isShallow === true
+				options.isShallow === true ||
+				property === '__ob__'
```

Comment:
Move `'__ob__'` out into a constant with a descriptive variable name.

Reply:
I'd rather see this as a configurable option, than a hard-coded constant. Non-Vue users who by coincidence use a property called `__ob__` will find this behavior very surprising. This is a general-purpose library: leaking details of other frameworks into the implementation sets an unsustainable precedent..

Reply:
@kpruden, I agree with making it an option. Two solutions come to mind: 
1. ignoreKeys: takes an array of key strings that should be skipped
2: ignoreUnderscore: Like ignoreSymbols, but ignores any property who's key begins with an underscore.

Reply:
@codenamezjames If you want to tackle one of these solutions, feel free, or I'll get to it soon.